From a35a1a5c11272ba1122cd85e13ff6a7b2f1ba8b9 Mon Sep 17 00:00:00 2001
From: Samuel Stirtzel <s.stirtzel@googlemail.com>
Date: Fri, 27 Jan 2012 08:17:03 +0100
Subject: [PATCH] Disable documentation. Signed-off-by: Samuel Stirtzel
 <s.stirtzel@googlemail.com>

---
 CMakeLists.txt |    7 +++++--
 1 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c82cdda..d91625b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,6 +6,9 @@ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules")
 # Make CPack available to easy generate binary packages
 include(CPack)
 
+## we need no docs, or at least an option to disable them!
+set (DISABLE_DOCS "Docs won't be build" TRUE)
+
 ################# set KDE specific information #################
 
 set (KDE_VERSION_MAJOR 4)
@@ -295,9 +298,9 @@ if (QT_QT3SUPPORT_FOUND)
 endif (QT_QT3SUPPORT_FOUND)
 add_subdirectory( kfile )
 add_subdirectory( kconf_update )
-if(NOT WINCE)
+if(NOT WINCE AND NOT DISABLE_DOCS)
 add_subdirectory( kdoctools  )
-endif(NOT WINCE)
+endif(NOT WINCE AND NOT DISABLE_DOCS)
 add_subdirectory( kioslave   )
 add_subdirectory( knewstuff  )
 add_subdirectory( kparts     )
-- 
1.7.5.4

