From 2f819210657590a5676b888bb8b4f1e3a41650a7 Mon Sep 17 00:00:00 2001
From: Samuel Stirtzel <s.stirtzel@googlemail.com>
Date: Thu, 2 Feb 2012 16:44:07 +0100
Subject: [PATCH] Fix Nepomuk Signed-off-by: Samuel Stirtzel
 <s.stirtzel@googlemail.com>

---
 CMakeLists.txt |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6600d8d..b27f979 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,4 +11,5 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_P
 add_definitions(-DQT_USE_FAST_CONCATENATION -DQT_USE_FAST_OPERATOR_PLUS)
 
 add_subdirectory(lib)
-add_subdirectory(service)
+
+#add_subdirectory(service)
-- 
1.7.5.4

