From bc1471806abc54cf2fd5b0604770b06d72220c8b Mon Sep 17 00:00:00 2001
From: Samuel Stirtzel <s.stirtzel@googlemail.com>
Date: Wed, 14 Mar 2012 11:20:46 +0100
Subject: [PATCH] Fix-FindKDE4Internals-cmake-file

Signed-off-by: Samuel Stirtzel <s.stirtzel@googlemail.com>
---
 cmake/modules/FindKDE4Internal.cmake |  153 +++++++++++++++++++---------------
 1 files changed, 84 insertions(+), 69 deletions(-)

diff --git a/cmake/modules/FindKDE4Internal.cmake b/cmake/modules/FindKDE4Internal.cmake
index 6d65e61..580840d 100644
--- a/cmake/modules/FindKDE4Internal.cmake
+++ b/cmake/modules/FindKDE4Internal.cmake
@@ -346,6 +346,8 @@
 # For details see the accompanying COPYING-CMAKE-SCRIPTS file.
 
 
+set(OPENEMBEDDED TRUE)
+
 # this is required now by cmake 2.6 and so must not be skipped by if(KDE4_FOUND) below
 cmake_minimum_required(VERSION 2.6.4 FATAL_ERROR)
 # set the cmake policies to the 2.4.x compatibility settings (may change for KDE 4.3)
@@ -521,10 +523,12 @@ if (_kdeBootStrapping)
       set(KDE4_MAKEKDEWIDGETS_EXECUTABLE    ${EXECUTABLE_OUTPUT_PATH}/${CMAKE_CFG_INTDIR}/makekdewidgets )
    else (WIN32)
       set(LIBRARY_OUTPUT_PATH               ${CMAKE_BINARY_DIR}/lib )
-      set(KDE4_KCFGC_EXECUTABLE             ${EXECUTABLE_OUTPUT_PATH}/${CMAKE_CFG_INTDIR}/kconfig_compiler${CMAKE_EXECUTABLE_SUFFIX}.shell )
-      set(KDE4_KAUTH_POLICY_GEN_EXECUTABLE  ${EXECUTABLE_OUTPUT_PATH}/${CMAKE_CFG_INTDIR}/kauth-policy-gen${CMAKE_EXECUTABLE_SUFFIX}.shell )
-      set(KDE4_MEINPROC_EXECUTABLE          ${EXECUTABLE_OUTPUT_PATH}/${CMAKE_CFG_INTDIR}/meinproc4${CMAKE_EXECUTABLE_SUFFIX}.shell )
-      set(KDE4_MAKEKDEWIDGETS_EXECUTABLE    ${EXECUTABLE_OUTPUT_PATH}/${CMAKE_CFG_INTDIR}/makekdewidgets${CMAKE_EXECUTABLE_SUFFIX}.shell )
+      if(NOT OPENEMBEDDED)
+	set(KDE4_KCFGC_EXECUTABLE             ${EXECUTABLE_OUTPUT_PATH}/${CMAKE_CFG_INTDIR}/kconfig_compiler${CMAKE_EXECUTABLE_SUFFIX}.shell )
+	set(KDE4_KAUTH_POLICY_GEN_EXECUTABLE  ${EXECUTABLE_OUTPUT_PATH}/${CMAKE_CFG_INTDIR}/kauth-policy-gen${CMAKE_EXECUTABLE_SUFFIX}.shell )
+	set(KDE4_MEINPROC_EXECUTABLE          ${EXECUTABLE_OUTPUT_PATH}/${CMAKE_CFG_INTDIR}/meinproc4${CMAKE_EXECUTABLE_SUFFIX}.shell )
+	set(KDE4_MAKEKDEWIDGETS_EXECUTABLE    ${EXECUTABLE_OUTPUT_PATH}/${CMAKE_CFG_INTDIR}/makekdewidgets${CMAKE_EXECUTABLE_SUFFIX}.shell )
+      endif(NOT OPENEMBEDDED)
    endif (WIN32)
 
    set(KDE4_LIB_DIR ${LIBRARY_OUTPUT_PATH}/${CMAKE_CFG_INTDIR})
@@ -582,8 +586,12 @@ else (_kdeBootStrapping)
 
    # KDE4_LIB_INSTALL_DIR and KDE4_INCLUDE_INSTALL_DIR are set in KDELibsDependencies.cmake,
    # use them to set the KDE4_LIB_DIR and KDE4_INCLUDE_DIR "public interface" variables
-   set(KDE4_LIB_DIR ${KDE4_LIB_INSTALL_DIR} )
-   set(KDE4_INCLUDE_DIR ${KDE4_INCLUDE_INSTALL_DIR} )
+   if(NOT KDE4_LIB_DIR)
+    set(KDE4_LIB_DIR ${KDE4_LIB_INSTALL_DIR} )
+   endif(NOT KDE4_LIB_DIR)
+   if(NOT KDE4_INCLUDE_DIR)
+    set(KDE4_INCLUDE_DIR ${KDE4_INCLUDE_INSTALL_DIR} )
+   endif(NOT KDE4_INCLUDE_DIR)
 
 
    # This setting is currently not recorded in KDELibsDependencies.cmake:
@@ -600,15 +608,20 @@ else (_kdeBootStrapping)
    list(GET _importedConfigurations 0 _firstConfig)
 
    if(NOT WINCE)
-   get_target_property(KDE4_KCFGC_EXECUTABLE             ${KDE4_TARGET_PREFIX}kconfig_compiler    LOCATION_${_firstConfig})
-   get_target_property(KDE4_MEINPROC_EXECUTABLE          ${KDE4_TARGET_PREFIX}meinproc4           LOCATION_${_firstConfig})
+    if(NOT KDE4_KCFGC_EXECUTABLE)
+      get_target_property(KDE4_KCFGC_EXECUTABLE             ${KDE4_TARGET_PREFIX}kconfig_compiler    LOCATION_${_firstConfig})
+    endif(NOT KDE4_KCFGC_EXECUTABLE)
+    if(NOT OPENEMBEDDED)
+      get_target_property(KDE4_MEINPROC_EXECUTABLE          ${KDE4_TARGET_PREFIX}meinproc4           LOCATION_${_firstConfig})
+    endif(NOT OPENEMBEDDED)
    else(NOT WINCE)
     set(KDE4_KCFGC_EXECUTABLE             ${HOST_BINDIR}/${CMAKE_CFG_INTDIR}/kconfig_compiler )
     set(KDE4_MEINPROC_EXECUTABLE          ${HOST_BINDIR}/${CMAKE_CFG_INTDIR}/meinproc4 )
    endif(NOT WINCE)
-   get_target_property(KDE4_KAUTH_POLICY_GEN_EXECUTABLE  ${KDE4_TARGET_PREFIX}kauth-policy-gen    LOCATION_${_firstConfig})
-   get_target_property(KDE4_MAKEKDEWIDGETS_EXECUTABLE    ${KDE4_TARGET_PREFIX}makekdewidgets      LOCATION_${_firstConfig})
-
+   if(NOT OPENEMBEDDED)
+    get_target_property(KDE4_KAUTH_POLICY_GEN_EXECUTABLE  ${KDE4_TARGET_PREFIX}kauth-policy-gen    LOCATION_${_firstConfig})
+    get_target_property(KDE4_MAKEKDEWIDGETS_EXECUTABLE    ${KDE4_TARGET_PREFIX}makekdewidgets      LOCATION_${_firstConfig})
+   endif(NOT OPENEMBEDDED)
    # allow searching cmake modules in all given kde install locations (KDEDIRS based)
    execute_process(COMMAND "${KDE4_KDECONFIG_EXECUTABLE}" --path data OUTPUT_VARIABLE _data_DIR ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
    file(TO_CMAKE_PATH "${_data_DIR}" _data_DIR)
@@ -784,49 +797,50 @@ set(LIB_SUFFIX "${_Init_LIB_SUFFIX}" CACHE STRING "Define suffix of directory na
 # this has to be after find_xxx() block above, since there KDELibsDependencies.cmake is included
 # which contains the install dirs from kdelibs, which are reused below
 
-if (WIN32)
+if (WIN32 OR OE_CROSSCOMPILING)
 # use relative install prefix to avoid hardcoded install paths in cmake_install.cmake files
 
-   set(LIB_INSTALL_DIR      "lib${LIB_SUFFIX}" )            # The subdirectory relative to the install prefix where libraries will be installed (default is ${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX})
+# _set_fancy will break install paths on OpenEmbedded
 
-   set(EXEC_INSTALL_PREFIX  "" )        # Base directory for executables and libraries
-   set(SHARE_INSTALL_PREFIX "share" )   # Base directory for files which go to share/
-   set(BIN_INSTALL_DIR      "bin"   )   # The install dir for executables (default ${EXEC_INSTALL_PREFIX}/bin)
-   set(SBIN_INSTALL_DIR     "sbin"  )   # The install dir for system executables (default ${EXEC_INSTALL_PREFIX}/sbin)
+   set(EXEC_INSTALL_PREFIX  "${CMAKE_INSTALL_PREFIX}" )        # Base directory for executables and libraries
+   set(SHARE_INSTALL_PREFIX "${EXEC_INSTALL_PREFIX}/share" )   # Base directory for files which go to share/
+   set(BIN_INSTALL_DIR      "${EXEC_INSTALL_PREFIX}/bin"   )   # The install dir for executables (default ${EXEC_INSTALL_PREFIX}/bin)
+   set(SBIN_INSTALL_DIR     "${EXEC_INSTALL_PREFIX}/sbin"  )   # The install dir for system executables (default ${EXEC_INSTALL_PREFIX}/sbin)
+   set(LIB_INSTALL_DIR      "${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX}" ) # The subdirectory relative to the install prefix where libraries will be installed (default is ${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX})
 
    set(LIBEXEC_INSTALL_DIR  "${BIN_INSTALL_DIR}"          ) # The subdirectory relative to the install prefix where libraries will be installed (default is ${BIN_INSTALL_DIR})
-   set(INCLUDE_INSTALL_DIR  "include"                     ) # The subdirectory to the header prefix
+   set(INCLUDE_INSTALL_DIR  "/usr/include"                     ) # The subdirectory to the header prefix
 
    set(PLUGIN_INSTALL_DIR       "lib${LIB_SUFFIX}/kde4"   ) #                "The subdirectory relative to the install prefix where plugins will be installed (default is ${LIB_INSTALL_DIR}/kde4)
    set(IMPORTS_INSTALL_DIR       "${PLUGIN_INSTALL_DIR}/imports"   ) # "The subdirectory relative to the install prefix where imports will be installed
-   set(CONFIG_INSTALL_DIR       "share/config"            ) # The config file install dir
-   set(DATA_INSTALL_DIR         "share/apps"              ) # The parent directory where applications can install their data
-   set(HTML_INSTALL_DIR         "share/doc/HTML"          ) # The HTML install dir for documentation
-   set(ICON_INSTALL_DIR         "share/icons"             ) # The icon install dir (default ${SHARE_INSTALL_PREFIX}/share/icons/)
-   set(KCFG_INSTALL_DIR         "share/config.kcfg"       ) # The install dir for kconfig files
-   set(LOCALE_INSTALL_DIR       "share/locale"            ) # The install dir for translations
-   set(MIME_INSTALL_DIR         "share/mimelnk"           ) # The install dir for the mimetype desktop files
-   set(SERVICES_INSTALL_DIR     "share/kde4/services"     ) # The install dir for service (desktop, protocol, ...) files
-   set(SERVICETYPES_INSTALL_DIR "share/kde4/servicetypes" ) # The install dir for servicestypes desktop files
-   set(SOUND_INSTALL_DIR        "share/sounds"            ) # The install dir for sound files
-   set(TEMPLATES_INSTALL_DIR    "share/templates"         ) # The install dir for templates (Create new file...)
-   set(WALLPAPER_INSTALL_DIR    "share/wallpapers"        ) # The install dir for wallpapers
-   set(DEMO_INSTALL_DIR         "share/demos"             ) # The install dir for demos
-   set(KCONF_UPDATE_INSTALL_DIR "share/apps/kconf_update" ) # The kconf_update install dir
-   set(AUTOSTART_INSTALL_DIR    "share/autostart"         ) # The install dir for autostart files
-
-   set(XDG_APPS_INSTALL_DIR      "share/applications/kde4"   ) # The XDG apps dir
-   set(XDG_DIRECTORY_INSTALL_DIR "share/desktop-directories" ) # The XDG directory
-   set(XDG_MIME_INSTALL_DIR      "share/mime/packages"       ) # The install dir for the xdg mimetypes
-
-   set(SYSCONF_INSTALL_DIR       "etc"                       ) # The sysconfig install dir (default /etc)
-   set(MAN_INSTALL_DIR           "share/man"                 ) # The man install dir (default ${SHARE_INSTALL_PREFIX}/man/)
-   set(INFO_INSTALL_DIR          "share/info"                ) # The info install dir (default ${SHARE_INSTALL_PREFIX}/info)")
-   set(DBUS_INTERFACES_INSTALL_DIR "share/dbus-1/interfaces" ) # The dbus interfaces install dir (default  ${SHARE_INSTALL_PREFIX}/dbus-1/interfaces)")
-   set(DBUS_SERVICES_INSTALL_DIR "share/dbus-1/services"     ) # The dbus services install dir (default  ${SHARE_INSTALL_PREFIX}/dbus-1/services)")
-   set(DBUS_SYSTEM_SERVICES_INSTALL_DIR "share/dbus-1/system-services"     ) # The dbus system services install dir (default  ${SHARE_INSTALL_PREFIX}/dbus-1/system-services)")
-
-else (WIN32)
+   set(CONFIG_INSTALL_DIR       "${SHARE_INSTALL_PREFIX}/config"            ) # The config file install dir
+   set(DATA_INSTALL_DIR         "${SHARE_INSTALL_PREFIX}/apps"              ) # The parent directory where applications can install their data
+   set(HTML_INSTALL_DIR         "${SHARE_INSTALL_PREFIX}/doc/HTML"          ) # The HTML install dir for documentation
+   set(ICON_INSTALL_DIR         "${SHARE_INSTALL_PREFIX}/icons"             ) # The icon install dir (default ${SHARE_INSTALL_PREFIX}/share/icons/)
+   set(KCFG_INSTALL_DIR         "${SHARE_INSTALL_PREFIX}/config.kcfg"       ) # The install dir for kconfig files
+   set(LOCALE_INSTALL_DIR       "${SHARE_INSTALL_PREFIX}/locale"            ) # The install dir for translations
+   set(MIME_INSTALL_DIR         "${SHARE_INSTALL_PREFIX}/mimelnk"           ) # The install dir for the mimetype desktop files
+   set(SERVICES_INSTALL_DIR     "${SHARE_INSTALL_PREFIX}/kde4/services"     ) # The install dir for service (desktop, protocol, ...) files
+   set(SERVICETYPES_INSTALL_DIR "${SHARE_INSTALL_PREFIX}/kde4/servicetypes" ) # The install dir for servicestypes desktop files
+   set(SOUND_INSTALL_DIR        "${SHARE_INSTALL_PREFIX}/sounds"            ) # The install dir for sound files
+   set(TEMPLATES_INSTALL_DIR    "${SHARE_INSTALL_PREFIX}/templates"         ) # The install dir for templates (Create new file...)
+   set(WALLPAPER_INSTALL_DIR    "${SHARE_INSTALL_PREFIX}/wallpapers"        ) # The install dir for wallpapers
+   set(DEMO_INSTALL_DIR         "${SHARE_INSTALL_PREFIX}/demos"             ) # The install dir for demos
+   set(KCONF_UPDATE_INSTALL_DIR "${SHARE_INSTALL_PREFIX}/apps/kconf_update" ) # The kconf_update install dir
+   set(AUTOSTART_INSTALL_DIR    "${SHARE_INSTALL_PREFIX}/autostart"         ) # The install dir for autostart files
+
+   set(XDG_APPS_INSTALL_DIR      "${SHARE_INSTALL_PREFIX}/applications/kde4"   ) # The XDG apps dir
+   set(XDG_DIRECTORY_INSTALL_DIR "${SHARE_INSTALL_PREFIX}/desktop-directories" ) # The XDG directory
+   set(XDG_MIME_INSTALL_DIR      "${SHARE_INSTALL_PREFIX}/mime/packages"       ) # The install dir for the xdg mimetypes
+
+   set(SYSCONF_INSTALL_DIR       "/etc"                       ) # The sysconfig install dir (default /etc)
+   set(MAN_INSTALL_DIR           "${SHARE_INSTALL_PREFIX}/man"                 ) # The man install dir (default ${SHARE_INSTALL_PREFIX}/man/)
+   set(INFO_INSTALL_DIR          "${SHARE_INSTALL_PREFIX}/info"                ) # The info install dir (default ${SHARE_INSTALL_PREFIX}/info)")
+   set(DBUS_INTERFACES_INSTALL_DIR "${SHARE_INSTALL_PREFIX}/dbus-1/interfaces" ) # The dbus interfaces install dir (default  ${SHARE_INSTALL_PREFIX}/dbus-1/interfaces)")
+   set(DBUS_SERVICES_INSTALL_DIR "${SHARE_INSTALL_PREFIX}/dbus-1/services"     ) # The dbus services install dir (default  ${SHARE_INSTALL_PREFIX}/dbus-1/services)")
+   set(DBUS_SYSTEM_SERVICES_INSTALL_DIR "${SHARE_INSTALL_PREFIX}/dbus-1/system-services"     ) # The dbus system services install dir (default  ${SHARE_INSTALL_PREFIX}/dbus-1/system-services)")
+
+else (WIN32 OR OE_CROSSCOMPILING)
 
    # This macro implements some very special logic how to deal with the cache.
    # By default the various install locations inherit their value from their "parent" variable
 
 
-- 
1.7.5.4

