--- a/configure.in	2012-02-15 11:37:14.686431813 +0100
+++ b/configure.in	2012-02-15 11:37:06.138432000 +0100
@@ -614,10 +614,10 @@
 CCDEBUG=""
 CCLIBS=""
 
-AC_CHECKING([default compiler values])
-if eval "${CC-cc} $CFLAGS $AM_LDFLAGS -o binsrc/config/ccdefs binsrc/config/ccdefs.c" ; then
-    eval `binsrc/config/ccdefs`
-fi
+#AC_CHECKING([default compiler values])
+#if eval "${CC-cc} $CFLAGS $AM_LDFLAGS -o binsrc/config/ccdefs binsrc/config/ccdefs.c" ; then
+#    eval `binsrc/config/ccdefs`
+#fi
 
 AC_ARG_WITH(debug, [dnl
 AC_HELP_STRING([--with-debug], [debugging code])dnl
@@ -856,64 +856,12 @@
  AC_MSG_ERROR([bad. Check config.log for details])
 ])
 
-AC_MSG_CHECKING([version and usability of the OpenSSL lib in ${openssl_dir}])
-AC_TRY_RUN([
-#include <openssl/rsa.h>
-#include <openssl/crypto.h>
-#include <openssl/x509.h>
-#include <openssl/pem.h>
-#include <openssl/ssl.h>
-#include <openssl/err.h>
-#include <openssl/bio.h>
-#include <openssl/asn1.h>
-#include <openssl/pkcs12.h>
-#include <openssl/rand.h>
-#include <stdio.h>
-#include <string.h>
-int
-main ()
-{
-    SSL_METHOD *ssl_server_method = NULL;
-    SSL_CTX *ssl_server_ctx = NULL;
-    unsigned char tmp[1024];
-
-    SSL_load_error_strings();
-    ERR_load_X509_strings();
-    ERR_load_PKCS12_strings();
-    #ifndef WIN32
-    RAND_bytes (tmp, sizeof(tmp));
-    RAND_add (tmp, sizeof(tmp), (double)(sizeof(tmp)));
-    #endif
-    SSLeay_add_ssl_algorithms();
-    PKCS12_PBE_add ();
-
-    ssl_server_method = SSLv23_client_method();
-    ssl_server_ctx = SSL_CTX_new (ssl_server_method);
-
-    if (!ssl_server_ctx)
-    {
-	ERR_print_errors_fp(stderr);
-	return(-1);
-    }
+BUILD_OPTS="$BUILD_OPTS ssl"
+ccincl="$ccincl $openssl_CFLAGS"
+srvrlibs="$srvrlibs $openssl_LDFLAGS"
+clntlibs="$clntlibs $openssl_LDFLAGS"
+AC_DEFINE(_SSL, 1, [OpenSSL])
 
-    if (OPENSSL_VERSION_NUMBER < 0x0090700f)
-    {
-	fprintf (stderr, "Wrong SSL version number\n");
-	return (-1);
-    }
-  return 0;
-}],
-[
- AC_MSG_RESULT(good)
- BUILD_OPTS="$BUILD_OPTS ssl"
- ccincl="$ccincl $openssl_CFLAGS"
- srvrlibs="$srvrlibs $openssl_LDFLAGS"
- clntlibs="$clntlibs $openssl_LDFLAGS"
- AC_DEFINE(_SSL, 1, [OpenSSL])
-],
-[
- AC_MSG_ERROR([bad. Check config.log for details])
-])
 LIBS="$save_LIBS"
 CPPFLAGS="$save_CPPFLAGS"
 AC_SUBST(openssl_CFLAGS)
