From 30b84c7f1903cb40ace738d039ee0fd374f3bc5c Mon Sep 17 00:00:00 2001
From: Samuel Stirtzel <s.stirtzel@googlemail.com>
Date: Wed, 29 Feb 2012 09:55:03 +0100
Subject: [PATCH] Opt out python scriptengine

Signed-off-by: Samuel Stirtzel <s.stirtzel@googlemail.com>
---
 plasma/generic/scriptengines/CMakeLists.txt |    9 +++++----
 1 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/plasma/generic/scriptengines/CMakeLists.txt b/plasma/generic/scriptengines/CMakeLists.txt
index a157f43..9ff702d 100644
--- a/plasma/generic/scriptengines/CMakeLists.txt
+++ b/plasma/generic/scriptengines/CMakeLists.txt
@@ -4,10 +4,11 @@ if(QT_QTWEBKIT_FOUND)
     macro_optional_add_subdirectory(webkit)
 endif(QT_QTWEBKIT_FOUND)
 
-if (PythonSupport_FOUND AND NOT WIN32)
-    macro_optional_add_subdirectory(python)
-endif (PythonSupport_FOUND AND NOT WIN32)
-
+if(NOT DEFINED WITHOUT_PYTHON)
+  if (PythonSupport_FOUND AND NOT WIN32)
+      macro_optional_add_subdirectory(python)
+  endif (PythonSupport_FOUND AND NOT WIN32)
+endif(NOT DEFINED WITHOUT_PYTHON)
 if(GOOGLEGADGETS_FOUND)
     macro_optional_add_subdirectory(google_gadgets)
 endif(GOOGLEGADGETS_FOUND)
-- 
1.7.5.4

