From 9f3d26b5cf198cad9df78358932ba1c6b7ebb54a Mon Sep 17 00:00:00 2001
From: Samuel Stirtzel <s.stirtzel@googlemail.com>
Date: Tue, 6 Mar 2012 09:58:15 +0100
Subject: [PATCH] Fix the KDE flavoured version of FindQt4.cmake

Signed-off-by: Samuel Stirtzel <s.stirtzel@googlemail.com>
---
 cmake/modules/FindQt4.cmake |    9 ++++++---
 1 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/cmake/modules/FindQt4.cmake b/cmake/modules/FindQt4.cmake
index f482d3c..e280eb2 100644
--- a/cmake/modules/FindQt4.cmake
+++ b/cmake/modules/FindQt4.cmake
@@ -456,9 +456,11 @@ FIND_PROGRAM(QT_QMAKE_EXECUTABLE NAMES qmake qmake4 qmake-qt4 qmake-mac PATHS
 
 IF (QT_QMAKE_EXECUTABLE)
 
-  IF(QT_QMAKE_EXECUTABLE_LAST)
-    STRING(COMPARE NOTEQUAL "${QT_QMAKE_EXECUTABLE_LAST}" "${QT_QMAKE_EXECUTABLE}" QT_QMAKE_CHANGED)
-  ENDIF(QT_QMAKE_EXECUTABLE_LAST)
+# This causes serious issues (that can't be tracked due to missing status messages) with cross compiling toolchains like OpenEmbedded
+  SET(QT_QMAKE_CHANGED FALSE)
+#  IF(QT_QMAKE_EXECUTABLE_LAST)
+#    STRING(COMPARE NOTEQUAL "${QT_QMAKE_EXECUTABLE_LAST}" "${QT_QMAKE_EXECUTABLE}" QT_QMAKE_CHANGED)
+#  ENDIF(QT_QMAKE_EXECUTABLE_LAST)
 
   SET(QT_QMAKE_EXECUTABLE_LAST "${QT_QMAKE_EXECUTABLE}" CACHE INTERNAL "" FORCE)
 
@@ -1059,6 +1061,7 @@ IF (QT4_QMAKE_FOUND)
 
 
   IF(QT_QMAKE_CHANGED)
+    MESSAGE(STATUS "Qt QMAKE has changed, so if this has happened KDE does some bogus checking, fix this please! Regards SST")
     SET(QT_UIC_EXECUTABLE NOTFOUND)
     SET(QT_MOC_EXECUTABLE NOTFOUND)
     SET(QT_UIC3_EXECUTABLE NOTFOUND)
-- 
1.7.5.4

