From 32f0055b402e6ddc2ae1ed7d8ea64c9bbaf372de Mon Sep 17 00:00:00 2001
From: Samuel Stirtzel <s.stirtzel@googlemail.com>
Date: Tue, 12 Jun 2012 16:11:53 +0200
Subject: [PATCH] Fix typo Namespace to NameSpace

Signed-off-by: Samuel Stirtzel <s.stirtzel@googlemail.com>
---
 conf/settings.kcfgc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/settings.kcfgc b/conf/settings.kcfgc
index 0b70390..7bec19b 100644
--- a/conf/settings.kcfgc
+++ b/conf/settings.kcfgc
@@ -1,5 +1,5 @@
 ClassName=Settings
-Namespace=Okular
+NameSpace=Okular
 File=okular.kcfg
 Mutators=true
 Singleton=true
-- 
1.7.9.5

